src/script/conversation/MessageRepository.ts

```diff
@@ -107,6 +107,7 @@ export interface MessageSendingOptions {
   /** Send native push notification for message. Default is `true`. */
   nativePush?: boolean;
   recipients?: QualifiedId[] | QualifiedUserClients | UserClients;
+  myClientsOnly?: boolean;
```

Comment:
I don't think this belongs here

Reply:
I think instead it should only be an extra param of the `sendCallingMessage` of the calling repo

Reply:
👍